app/views/calendars/2020-01.html.erb

```diff
@@ -0,0 +1,7 @@
+    <h1>NAICS Bar Chart Race</h2>
+    <div class="naics-bar-chart-race"></div>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/5.14.2/d3.min.js"></script>
```

Comment:
I think you should be able to add [d3 with yarn](https://yarnpkg.com/en/package/d3) in `naics-bar-chart-race.js`. Try it:

```
yarn add d3
```

Then add the line 
```
import * as d3 from "d3";
```
to `naics-bar-chart-race.js`.